crates/fluvio-storage/src/replica.rs

```diff
@@ -40,7 +40,7 @@ pub struct FileReplica {
     commit_checkpoint: CheckPoint,
     cleaner: Arc<Cleaner>,
     size: Arc<ReplicaSize>,
-    append_failure: bool, // if this is true, last append failed, should not append again
+    error_msg: Option<String>,
```

Comment:
Error message can be recovered from log, so there is no need hold on this. Beside which error?

Reply:
Same comment as @nacardin,  in the SPU, if there are certain number of errors happen,  entire replica should be disabled and return offline status